Catch me

Usage: exceptionFromCommand [Arguments]

Arguments:
catcher 
